test/e2e/autoscale_test.go

```diff
@@ -102,6 +102,11 @@ func TestAutoscaleSustaining(t *testing.T) {
 		autoscaling.MetricAggregationAlgorithmWeightedExponential,
 	} {
 		algo := algo
+		if testing.Short() && algo == autoscaling.MetricAggregationAlgorithmWeightedExponential {
```

Comment:
I've seen linear be flakey as well - can we skip the entire `TestAutoscaleSustaining` if we're requesting `short`